addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -42,7 +42,7 @@
                                 </div>
                             </div>
                             <br/>
-                            <h1 t-field="o.name" class="mt0 float-start"/>
+                            <h1 class="mt-4" t-field="o.name"/>
```

Comment:
Why change the mt0 part of the class?
```suggestion
                            <h1 t-field="o.name" class="mt0"/>
```

Reply:
@ticodoo i have updated the pr title and commit message and mt4 class has changed back to mto as in source code.